app/bettermail.go

```diff
@@ -127,6 +135,72 @@ func handlePushPayload(payload PushPayload, c appengine.Context) (*mail.Message,
 	return message, nil
 }
 
+func fetchCommit(payload CommitCommentPayload, commit *ApiCommit, c appengine.Context) error {
+	commitSHA := *payload.Comment.CommitID
+	url := strings.Replace(*payload.Repo.CommitsURL, "{/sha}", "/" + commitSHA, 1)
+
+	client := urlfetch.Client(c)
+	resp, err := client.Get(url)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+	return json.NewDecoder(resp.Body).Decode(commit)
+}
+
+func handleCommitCommentPayload(payload CommitCommentPayload, c appengine.Context) (*mail.Message, error) {
+	var commit ApiCommit
+	title := ""
+	err := fetchCommit(payload, &commit, c)
+	if err != nil {
+		c.Infof("Error fetching commit = %v\n", err)
+	} else {
+		title, _ = getTitleAndMessageFromCommitMessage(*commit.Commit.Message)
+	}
+
+	// TODO: allow location to be customized
+	location, _ := time.LoadLocation("America/Los_Angeles")
+	updatedDate := payload.Comment.UpdatedAt.In(location)
+
+	commitShortSHA := (*payload.Comment.CommitID)[:7]
+
+	var data = map[string]interface{}{
+		"Payload":                  payload,
+		"Comment":                  payload.Comment,
+		"Sender":                   payload.Sender,
+		"Repo":                     payload.Repo,
+		"ShortSHA":                 commitShortSHA,
+		"Body":                     *payload.Comment.Body,
+		"CommitURL":                commit.HTML_URL,
+		"UpdatedDisplayDate":       safeFormattedDate(updatedDate.Format(DisplayDateFormat)),
+	}
+
+	var mailHtml bytes.Buffer
+	if err := templates["commit-comment"].Execute(&mailHtml, data); err != nil {
+		return nil, err
+	}
+
+	senderUserName := *payload.Sender.Login
+	senderName := senderUserName
+
+	sender := fmt.Sprintf("%s <%s@better-github-mail.appspotmail.com>", senderName, senderUserName)
+	subject := fmt.Sprintf("Re: [%s] %s: %s", *payload.Repo.FullName, commitShortSHA, title)
+	c.Infof("subject = %s\n", subject)
+
+	recipient := "eng+commits@quip.com"
+	if appengine.IsDevAppServer() {
+		recipient = "mihai@quip.com"
+	}
+
+	message := &mail.Message{
```

Comment:
Do you get the message ID after sending? We could persist that along with the subject.